cloudigrade/config/settings/base.py

```diff
@@ -153,6 +153,6 @@
 
 # Message and Task Queues
 
-RABBITMQ_EXCHANGE_NAME = 'cloudigrade_inspectigrade'
-RABBITMQ_QUEUE_NAME = 'machine_images'
-RABBITMQ_URL = 'amqp://guest:guest@localhost:5672/%2F'
+RABBITMQ_EXCHANGE_NAME = env('RABBITMQ_EXCHANGE_NAME', default='cloudigrade_inspectigrade')
+RABBITMQ_QUEUE_NAME = env('RABBITMQ_QUEUE_NAME', default='machine_images')
+RABBITMQ_URL = env('RABBITMQ_URL', default='amqp://guest:guest@localhost:5672/%2F')
```

Comment:
This reminds me. I replaced this with `CELERY_BROKER_URL` in my branch, but I suppose we need both since we will have standard message queues via Rabbit directly.